src/Components/ui/breadcrumb.tsx

```diff
@@ -0,0 +1,118 @@
+import * as React from "react";
```

Comment:
The folder casing for the file is wrong.

Reply:
Can you explain @bodhish

Reply:
my  folder src/Components/ui/breadcrumb.tsx, but real folder changed to components (case changed), how can i update that changes in my vsCode, when i chainge code and try git status it shows changed src/Components/ui/breadcrumb.tsx  @bodhish

Reply:
i cant change the folder path, i try to pull the code, but didnt work